README.md

```diff
@@ -74,7 +74,6 @@ Welcome to the first, open-sourced, structured, peer-reviewed threat modeling da
 
    ```shell
    $ check-jsonschema --schemafile threat-model.schema.json threat-models/husky-ai-threat-model.json 
-   ok -- validation done
```

Comment:
@P3tra-WP, FYI, this line 77 was meant to represent the expected output of the command. But it wasn't really important to keep it.